frontend/utilities/strings/stringUtils.ts

```diff
@@ -79,9 +79,25 @@ export const strToBool = (str?: string | null) => {
   return str ? JSON.parse(str) : false;
 };
 
+export const stripQuotes = (string: string) => {
+  // Regular expression to match quotes at the start and end of the string
+  const quoteRegex = /^["''""]+([\s\S]*?)["''""]$/;
+
+  // If the string matches the regex, return the content between the quotes
+  // Otherwise, return the original string
+  return quoteRegex.test(string) ? string.replace(quoteRegex, "$1") : string;
+};
+
+export const isIncompleteQuoteQuery = (str: string) => {
+  const pattern = /^(['"])(?!.*\1$)/;
```

Comment:
Do we want this regex to match both single and double quotes? This matches both. The Figma seems to call for doubles only, but maybe there was some other discussion around it?